src/dolphin/workflows/config/_common.py

```diff
@@ -104,6 +104,13 @@ class PhaseLinkingOptions(BaseModel, extra="forbid"):
         gt=0.0,
         lt=1.0,
     )
+    baseline_lag: Optional[int] = Field(
+        None,
+        description=(
```

Comment:
```suggestion
        None,
        gt=0,
        description=(
```

(is this right? it has to be larger than 0 if passed?)